packages/lambdas/src/fhir-to-bundle.ts

```diff
@@ -0,0 +1,86 @@
+import { Resource } from "@medplum/fhirtypes";
+import { medical } from "@metriport/shared";
+import { SearchSetBundle } from "@metriport/shared/medical";
+import { Patient } from "@metriport/core/domain/patient";
+import { getConsolidatedFhirBundle } from "@metriport/core/external/fhir/consolidated";
+import { out } from "@metriport/core/util/log";
+import * as Sentry from "@sentry/serverless";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+import { apiClient } from "./shared/oss-api";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const lambdaName = getEnvOrFail("AWS_LAMBDA_FUNCTION_NAME");
+const apiURL = getEnvOrFail("API_URL");
+
+const ossApi = apiClient(apiURL);
+
+export const handler = Sentry.AWSLambda.wrapHandler(
+  async ({
```

Comment:
re: avoiding duplicated Sentry events
We moved some lambdas from using Sentry's default handler so we can avoid duplicated events.
We usually want to have our own `capture.error` on a global error handling code in the lambda so we can send more useful info to Sentry, to help us debug issues.
I'd consider removing the default handler and having just the global handler (with the conditional in case of async=false, below).